server/pages/index.js

```diff
@@ -80,7 +81,11 @@ function generateStaticUrls( request ) {
 		urls[ file.path ] = getUrl( file.path, file.hash );
 	} );
 
-	assets = request.app.get( 'assets' );
+	// vendor dll
+	urls.vendor = urls[ 'vendor.' + config( 'env' ) + '.js' ];
```

Comment:
Any reason not to do the lookup by env in the Jade template instead of aliasing it here? We're already doing similar for `jsFile`, e.g.

```jade
script(src=urls[ 'vendor.' + env + '.js' ])
```